component/http/cache/cache_model.go

```diff
@@ -0,0 +1,68 @@
+package cache
```

Comment:
since we are in the cache package could we name the file without the `cache_prefix`.